src/components/learner-credit-management/AssignmentRowActionTableCell.jsx

```diff
@@ -1,45 +1,40 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import {
-  Icon,
-  IconButton,
-  OverlayTrigger,
-  Stack,
-  Tooltip,
+  Icon, IconButton, OverlayTrigger, Stack, Tooltip,
 } from '@edx/paragon';
 import { Mail, DoNotDisturbOn } from '@edx/paragon/icons';
 
 const AssignmentRowActionTableCell = ({ row }) => {
-  const cancelButtonMarginLeft = row.original.state === 'allocated' ? 'ml-2.5' : 'ml-auto';
+  const learnerStateBool = row.original.learnerState === 'waiting';
+  const cancelButtonMarginLeft = learnerStateBool ? 'ml-2.5' : 'ml-auto';
   return (
-    <div className="d-flex">
-      {row.original.state === 'allocated' && (
-        <>
-          <OverlayTrigger
-            key="Remind"
-            placement="top"
-            overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
-          >
-            <IconButton
-              className="ml-auto mr-0"
-              src={Mail}
-              iconAs={Icon}
-              alt="Remind learner"
+    <Stack direction="horizontal" gap="1">
+      {learnerStateBool && (
+      <OverlayTrigger
+        key="Remind"
+        placement="top"
+        overlay={<Tooltip variant="light" id="tooltip-remind">Remind learner</Tooltip>}
```

Comment:
Based on the Figma spec and the Subscription Management table (which is what we're largely working against for consistency), the `Tooltip`'s variant should be the default such that the background is the dark color, as opposed to white.

<img width="132" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/c8930f15-fb9e-4667-8aa2-30f2c7f0fecf">

<br />
<br />

<img width="1434" alt="image" src="https://github.com/openedx/frontend-app-admin-portal/assets/2828721/0e9d190a-53c7-4e5b-a535-286b16c7bfff">